### PR TITLE
Optimize diagram

### PR DESCRIPTION
**Pull request by NguyenXuanTungB**

### CODE REVIEW

File: app/src/main/res/layout/activity_choose_floor.xml [hunk 1]
```diff
@@ -23,39 +23,47 @@
...
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="right|top"
-                    android:layout_marginTop="20dp"
-                    android:padding="10dp"
-                    android:paddingRight="10dp"
+                    android:layout_marginTop="@dimen/common_size_20"
+                    android:padding="@dimen/common_size_10"
+                    android:paddingRight="@dimen/common_size_10"
                     android:text="find way" />
```
Comment (nguyenvantoan20111994) [inline app/src/main/res/layout/activity_choose_floor.xml]: hardcode

File: app/src/main/res/layout/activity_choose_floor.xml [hunk 2]
```diff
@@ -23,39 +23,47 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="right|top"
-                    android:layout_marginTop="20dp"
-                    android:padding="10dp"
-                    android:paddingRight="10dp"
+                    android:layout_marginTop="@dimen/common_size_20"
+                    android:padding="@dimen/common_size_10"
+                    android:paddingRight="@dimen/common_size_10"
                     android:text="make edge" />
```
Comment (nguyenvantoan20111994) [inline app/src/main/res/layout/activity_choose_floor.xml]: hardcode

File: app/src/main/res/layout/activity_choose_floor.xml [hunk 3]
```diff
@@ -23,39 +23,47 @@
...
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="right|top"
-                    android:layout_marginTop="20dp"
-                    android:padding="10dp"
-                    android:paddingRight="10dp"
+                    android:layout_marginTop="@dimen/common_size_20"
+                    android:padding="@dimen/common_size_10"
+                    android:paddingRight="@dimen/common_size_10"
                     android:text="delete all" />
```
Comment (nguyenvantoan20111994) [inline app/src/main/res/layout/activity_choose_floor.xml]: hardcode

File: app/src/main/res/layout/activity_choose_floor.xml [hunk 4]
```diff
@@ -23,39 +23,47 @@
...
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="right|top"
-                    android:layout_marginTop="20dp"
-                    android:padding="10dp"
-                    android:paddingRight="10dp"
+                    android:layout_marginTop="@dimen/common_size_20"
+                    android:padding="@dimen/common_size_10"
+                    android:paddingRight="@dimen/common_size_10"
                     android:text="draw path" />
```
Comment (nguyenvantoan20111994) [inline app/src/main/res/layout/activity_choose_floor.xml]: hardcode

File: app/src/main/java/com/example/framgia/imarketandroid/util/algorithm/DijkstraAlgorithm.java [hunk 5]
```diff
@@ -31,23 +32,13 @@
     private Map<Point, Float> mDistance;
     private Context context;
     private RealmRemote realmRemote;
```
Comment (nguyenvantoan20111994) [inline app/src/main/java/com/example/framgia/imarketandroid/util/algorithm/DijkstraAlgorithm.java]: convention

**Comment by TranTrungPhong** [comment]
OK

**Pull request merged**